sympy/physics/vector/point.py

```diff
@@ -480,22 +480,36 @@ def vel(self, frame):
         frame : ReferenceFrame
             The frame in which the returned velocity vector will be defined in
 
+        otherpoint : Point
+            The point w.r.t which self's position is defined, works as initial
+            point.
+
         Examples
         ========
 
-        >>> from sympy.physics.vector import Point, ReferenceFrame
+        >>> from sympy.physics.vector import Point, ReferenceFrame, dynamicsymbols
         >>> N = ReferenceFrame('N')
         >>> p1 = Point('p1')
         >>> p1.set_vel(N, 10 * N.x)
         >>> p1.vel(N)
         10*N.x
+        >>> q = dynamicsymbols('q')
+        >>> p2 = Point('p2')
+        >>> p2.set_pos(p1, q*N.x)
+        >>> p2.vel(N, otherpoint = p1)
+        Derivative(q(t), t)*N.x
```

Comment:
I'm a bit confused - is this any different from `p2.vel() - p1.vel()`?

Reply:
Yes alot , it's not calculating relative velocity , the idea was that a point shall give velocity if proper displacement vectors are present , so the function calculates veloctiy of point p2  wrt  to point 1 , not relative velocity , I think i shall change the example in docstring

Reply:
> veloctiy of point p2 wrt to point 1 , not relative velocity

"relative to" and "with respect to" are interchangeable here in my opinion.

I think the answer to my question is that while this _is_ geometrically equivalent to `p2.vel(N) - p1.vel(N)`, `p2.vel(N)` raises an exception so that spelling isn't possible.

Reply:
@eric-wieser Check the docstring example now , its more clearer, the changes I did would calculate velocity of given point from the position(displacement) vector if the velocity dictionary does not contain value for that reference frame.
As said by @moorepants https://github.com/sympy/sympy/issues/17761#issuecomment-687156490

Reply:
> I'm a bit confused - is this any different from `p2.vel(N) - p1.vel(N)`?

It differentiates displacement vector to get velocity vector , to get displacement it takes otherpoint